Automattic/wp-calypso
# Change the design of the Filters section in the domain suggestions screen

#### Changes proposed in this Pull Request

* Add option to have labels in the suggestion list in `TokenField`
* Add labels for "Most popular endings" in the TLD filters list in the domain suggestions screen
* Change the `Filters` button design in the domain suggestions page

The new filter button looks like this:
<img width="1066" alt="Screenshot 2021-07-30 at 14 12 08" src="https://user-images.githubusercontent.com/1355045/127645063-c13faa09-091a-4563-89e9-7a250bbc619d.png">
<img width="413" alt="Screenshot 2021-07-30 at 14 12 15" src="https://user-images.githubusercontent.com/1355045/127645067-3b3bae72-629b-4f3d-a1a6-333fb936c99f.png">
<img width="414" alt="Screenshot 2021-07-30 at 14 12 32" src="https://user-images.githubusercontent.com/1355045/127645068-db6b23cb-3c73-44fc-9110-7ec2a0a450bc.png">
<img width="1067" alt="Screenshot 2021-07-30 at 14 12 40" src="https://user-images.githubusercontent.com/1355045/127645070-e4fa756c-78b9-4fcc-8b17-ee40e1e13ed0.png">


The `TokenField` component with labels between the options looks like this (you can check it in the component examples in /devdocs/design/token-fields):
<img width="428" alt="Screenshot 2021-07-29 at 22 39 27" src="https://user-images.githubusercontent.com/1355045/127555487-ff4aa094-87b5-45cc-bb95-8ffa74560078.png">
<img width="419" alt="Screenshot 2021-07-29 at 22 39 35" src="https://user-images.githubusercontent.com/1355045/127555497-61c6cf82-f6ea-4b0e-8f8e-cbd501f258ea.png">

#### Testing instructions

* Visit /devdocs/design/token-fields and check the "TokenField with Expanded Suggestions and Labels" section
